apps/hash-ai-worker-ts/turbo.json

```diff
@@ -3,6 +3,9 @@
   "tasks": {
     "build:docker": {
       "cache": false
+    },
+    "build": {
+      "outputs": ["./dist/**"]
     }
```

Comment:
We should watch out for the turborepo remote cache if we frequently cache the `dist` folder.